service/src/main/java/bio/terra/workspace/service/resource/ValidationUtils.java

```diff
@@ -116,6 +131,48 @@ public static void validateBucketName(String name) {
     }
   }
 
+  /** Validate whether the input URI is a valid GitHub Repo https uri. */
```

Comment:
nit: I think the comment is not quite right - it doesn't have to be https, right?